azure/services/aso/aso_test.go

```diff
@@ -710,7 +710,7 @@ func TestCreateOrUpdateResource(t *testing.T) {
 
 		specMock.MockTagsGetterSetter.EXPECT().GetActualTags(gomock.Any()).Return(nil)
 		specMock.MockTagsGetterSetter.EXPECT().GetAdditionalTags().Return(nil)
-		specMock.MockTagsGetterSetter.EXPECT().GetDesiredTags(gomock.Any()).Return(nil)
+		specMock.MockTagsGetterSetter.EXPECT().GetDesiredTags(gomock.Any()).Return(nil).Times(2)
```

Comment:
That means this will be called twice or else the test should fail. gomock complains if no count is specified this way and the function is called more than once. Whether we specify `.Times(2)` or `.AnyTimes()` doesn't matter to me since the "2" isn't significant other than to get the test to pass.